sys/include/phydat.h

```diff
@@ -98,6 +98,7 @@ enum {
     /* pressure */
     UNIT_BAR,       /**< Beer? */
     UNIT_PA,        /**< Pascal */
+    UNIT_HPA,       /**< hectoPascal */
```

Comment:
yes, but SAUL does not allow to scale the unit (that is add a prefix) **and** scale the value separately, i.e. its not possible to have something like `1013.25hPa`. At least not without introducing an explicit `UNIT_HPA` as suggested here. 

But, I was asking for opinions here, primarily - 'cause I'm unsure whether such is needed or not. I kind of have a use-case for lately reworked pressure sensor drivers, namely bmp180, bmx280 and mpl3115a.